OPHD/UI/Core/RadioButton.cpp

```diff
@@ -1,30 +1,23 @@
-#include "RadioButton.h"
-
-#include "UIContainer.h"
-#include "../../Cache.h"
-#include "../../Constants.h"
-
-#include <NAS2D/Utility.h>
-#include <NAS2D/Renderer/Renderer.h>
-#include <NAS2D/MathUtils.h>
-
-#include <algorithm>
-
+#include "RadioButtonGroup.h"
 
 using namespace NAS2D;
 
-
-RadioButton::RadioButton(std::string newText) :
+RadioButtonGroup::RadioButton::RadioButton(std::string newText, RadioButtonGroup* parentContainer, NAS2D::DelegateX<void> delegate) :
```

Comment:
I would probably try to group the `newText` and `delegate` parameters next to each other, as they are kind of interrelated. The `parentContainer` parameter is more standalone.

Reply:
Moved parentContainer to the front.